github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml

```diff
@@ -54,3 +54,58 @@ periodics:
       resources:
         requests:
           memory: "8Gi"
+- name: periodic-common-templates-nightly-centos-container-disk-image-builder
+  cron: "30 2 * * *"
```

Comment:
Could you please move the trigger time one hour later? So that it doesn't match the time of the other periodic.

Reply:
Done